mono/utils/mono-rand.c

```diff
@@ -93,19 +93,19 @@ mono_getrandom (guchar *buffer, gint buffer_size, gint flags, MonoError *error)
    - Return -1 on error
    getentropy() is retried if it failed with EINTR: interrupted by a signal. */
 static gint
-mono_getentropy (guchar *buffer, gint buffer_size, MonoError *error)
+mono_getentropy (guchar *buffer, intptr_t buffer_size, MonoError *error)
 {
 	static gint getentropy_works = 1;
 	gint count = 0;
 	gint len;
 	gint err;
 
+	error_init (error);
```

Comment:
Remove this `error_init`. `mono_getentropy` is only called by `mono_rand_try_get_bytes` which already calls `error_init`.